packages/web-react/src/components/CheckboxField/useCheckboxFieldStyleProps.tsx

```diff
@@ -16,11 +16,12 @@ export interface CheckboxFieldStyles {
 }
 
 export function useCheckboxFieldStyleProps(props: SpiritCheckboxFieldProps): CheckboxFieldStyles {
-  const { validationState, isLabelHidden, ...restProps } = props;
+  const { validationState, isItem, isLabelHidden, ...restProps } = props;
   const { isDisabled, isRequired } = restProps;
 
   const checkboxFieldClass = useClassNamePrefix('CheckboxField');
   const checkboxFieldDisabledClass = `${checkboxFieldClass}--disabled`;
+  const checkboxFieldItemClass = `${checkboxFieldClass}--isItem`;
```

Comment:
I think that this is weird. We have a `disabled` modifier, not `isDisabled`, we have `error` modifier not `hasError`. Can we be more consistent or is there something else behind it that I am not seeing yet?

Reply:
ok, I unified it to `--item`.